Modules/TRD/TRDQC.json

```
@@ -0,0 +1,144 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "505618",
+        "type": "2",
+        "periodName": "",
+        "passName": "",
+        "provenance": "qc"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
```

The url for consul should be left empty. 
```suggestion
        "url": ""
```

---

Ah shot, was never what was supposed to go in that stuff in anycase ;-)